.github/workflows/test.yml

```diff
@@ -13,13 +13,19 @@ jobs:
   build_and_test_ubuntu:
     runs-on: ubuntu-latest
     container:
-      image: rostooling/setup-ros-docker:ubuntu-jammy-latest
+      image: rostooling/setup-ros-docker:ubuntu-noble-latest
     steps:
     - uses: ros-tooling/action-ros-ci@v0.3
       with:
         package-name: libstatistics_collector
-        colcon-mixin-name: coverage-gcc
-        colcon-mixin-repository: https://raw.githubusercontent.com/colcon/colcon-mixin-repository/master/index.yaml
+        colcon-defaults: |
+          {
+            "build": {
+              "mixin": [
+                "coverage-gcc"
+              ]
+            }
+          }
         target-ros2-distro: rolling
```

Comment:
Oh wait. I realized that you backported the commit from #193. Can you change `rolling` to `jazzy` here?

Reply:
Thank you for your suggestion. I changed the distro.